build/l10n-pseudo.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
```

Comment:
I'd say `build/l10n.js` *is* related to the buildsystem: it's a set of modifications on top of `shared/js/l10n.js` which make it possible to run the buildtime l10n logic, like `GAIA_CONCAT_LOCALES` and `GAIA_PRETRANSLATE`.

For instance on runtime, when `shared/js/l10n.js` reads a *properties file in, it parses it into an AST and creates a dict of Entity objects out of it, quickly discarding the AST.  On buildtime however, we keep the AST around so that we're able to serialize it into JSON files when `GAIA_CONCAT_LOCALES` is 1.